orca/orca-retrofit/src/main/groovy/com/netflix/spinnaker/orca/retrofit/exceptions/BaseRetrofitExceptionHandler.groovy

```diff
@@ -37,10 +34,6 @@ abstract class BaseRetrofitExceptionHandler implements ExceptionHandler {
       return true
     }
 
-    if(httpMethod == null) {
```

Comment:
I think `boolean shouldRetry(Exception e, String kind, Integer responseCode)` needs to disappear.  I don't think it's used, but it no longer makes sense to have httpMethod null here.

Reply:
removed in https://github.com/spinnaker/spinnaker/pull/7173/commits/b7ad47544c3603d2a2d5d02fa07ec6a282d03aa4